projects/plugins/jetpack/class.jetpack-connection-banner.php

```diff
@@ -251,91 +267,60 @@ public static function enqueue_connect_button_scripts() {
-					?>
-
+			<?php
+			if ( ! $this->force_display() ) :
+				?>
 					<span
-						class="notice-dismiss connection-banner-dismiss"
+						class="notice-dismiss jp-connection-banner__dismiss"
 						title="<?php esc_attr_e( 'Dismiss this notice', 'jetpack' ); ?>">
 					</span>
 
 					<?php
 				endif;
-				?>
-
-				<div class="jp-wpcom-connect__content-container">
-
-					<!-- slide 1: intro -->
-					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
-
-						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
-							<?php
-							$logo = new Logo();
-							echo $logo->render(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Returns SVG.
-							?>
-							<img
-								src="<?php echo esc_url( plugins_url( 'images/jetpack-powering-up.svg', JETPACK__PLUGIN_FILE ) ); ?>"
-								class="jp-wpcom-connect__hide-phone-and-smaller"
-								alt="
-								<?php
-								esc_attr_e(
-									'Jetpack premium services offer even more powerful performance, security, and revenue tools to help you keep your site safe, fast, and help generate income.',
-									'jetpack'
-								);
-								?>
-								"
-								height="auto"
-								width="225"
-								/>
-						</div>
-
-						<div class="jp-wpcom-connect__slide-text">
-							<h2><?php esc_html_e( 'Simplify your site security and performance with Jetpack', 'jetpack' ); ?></h2>
-
-							<p>
-								<?php
-								esc_html_e(
-									'Jetpack protects you against brute force attacks and unauthorized logins. Basic protection is always free, while premium plans add unlimited backups of your whole site, spam protection, malware scanning, and automated fixes.',
-									'jetpack'
-								);
-								?>
-							</p>
-
-							<p>
-								<?php
-								esc_html_e(
-									'Activate site accelerator tools and watch your page load times decrease—we’ll optimize your images and serve them from our own powerful global network of servers, and speed up your mobile site to reduce bandwidth usage.',
-									'jetpack'
-								);
-								?>
-							</p>
-
-							<div class="jp-banner__button-container">
-								<span class="jp-banner__tos-blurb"><?php jetpack_render_tos_blurb(); ?></span>
-								<a
-										href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>"
-										class="dops-button is-primary jp-banner__alt-connect-button">
-									<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
-								</a>
-							</div>
-
-						</div>
-					</div> <!-- end slide 1 -->
+			?>
+			<div class="jp-connection-banner__inner">
+				<div class="jp-connection-banner__content">
+					<div class="jp-connection-banner__logo">
+						<?php echo $jetpack_logo->get_jp_emblem_larger(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+					</div>
+					<h2 class="jp-connection-banner__title"><?php esc_html_e( 'Simplify your site security and performance with&nbsp;Jetpack', 'jetpack' ); ?></h2>
```

Comment:
```suggestion
					<h2 class="jp-connection-banner__title"><?php esc_html_e( 'Simplify your site security and performance with Jetpack', 'jetpack' ); ?></h2>
```

Is there a particular reason for using `&nbsp;` here?

Reply:
I just wanted to avoid [text widows](https://en.wikipedia.org/wiki/Widows_and_orphans) that happen in specific edge cases, but indeed it's not a good option to put it in the translation.